doc/source/serve/deployment-graph/chain_nodes_same_class_different_args.md

```diff
@@ -0,0 +1,54 @@
+# Pattern: Chain nodes with same class and different args
+
+The example shows how to chain nodes using the same class and having different args passed in
+
+![pic](chain_nodes_same_class_different_args.svg)
+
+## Code
+
++++
+
```

Comment:
Can we please use `literalinclude` from a CI-tested Python file in e.g. a `doc_test` folder. We can't afford the maintenance burden of untested snippets.